packages/slate-plugins/src/__tests__/serializers/serialize-html/htmlSerialize.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { htmlSerialize } from 'serializers/serialize-html';
+import {
+  BoldPlugin,
+  HighlightPlugin,
+  ItalicPlugin,
+  StrikethroughPlugin,
+  CodePlugin,
+  SubscriptPlugin,
+  SuperscriptPlugin,
+  UnderlinePlugin,
+  LinkPlugin,
+  ListPlugin,
+} from '../../..';
+
+it('serialize bold to html', () => {
+  expect(
+    htmlSerialize([BoldPlugin()])([
+      { text: 'Some paragraph of text with ' },
+      { text: 'bold', bold: true },
+      { text: ' part.' },
+    ])
+  ).toEqual('Some paragraph of text with <strong>bold</strong> part.');
+});
+
+it('serialize italic to html', () => {
+  expect(
+    htmlSerialize([ItalicPlugin()])([
+      { text: 'Some paragraph of text with ' },
+      { text: 'italic', italic: true },
+      { text: ' part.' },
+    ])
+  ).toEqual('Some paragraph of text with <em>italic</em> part.');
+});
+
+it('serialize highlight to html', () => {
+  expect(
+    htmlSerialize([HighlightPlugin()])([
+      { text: 'Some paragraph of text with ' },
+      { text: 'highlighted', highlight: true },
+      { text: ' part.' },
+    ])
+  ).toEqual(
+    'Some paragraph of text with <span class="sc-fznWqX jhjZFC">highlighted</span> part.'
+    // THIS SHOULD LOOK LIKE THIS:
+    // 'Some paragraph of text with <mark>highlighted</mark> part.'
+  );
+});
+
+it('serialize strikethrough to html', () => {
+  expect(
+    htmlSerialize([StrikethroughPlugin()])([
+      { text: 'Some paragraph of text with ' },
+      { text: 'strikethrough', strikethrough: true },
+      { text: ' part.' },
+    ])
+  ).toEqual(
+    'Some paragraph of text with <span class="sc-fzqARJ jiBykp">strikethrough</span> part.'
+    // We need to add explicit class to this span 'strikethrough'
+  );
+});
+
+it.skip('serialize code to html', () => {
```

Comment:
Not sure why, but the code plugin does not render anything for me here. Need to do more investigation.